README.md

```diff
@@ -42,6 +46,13 @@ After generating the configure script with ``autogen``,  ``./configure --help``
 - ``make``
 - ``sudo make install``
 
+Run unit tests:
+
+- ``make check``
+
+Note that if you configured PointCloud using a non-standard LibGHT location, you may need to add its location to the ``LD_LIBRARY_PATH`` environment variable. For example:
```

Comment:
This should not be needed IFF test binaries are created via libtool (which produces scripted wrappers to do just this).
Do you confirm this is not the case for our tests ? And, does this only apply to the autotools method ?